### PR TITLE
AB: Adappt Email Notification Update

### PR DESCRIPTION
**Pull request by morgang-adappt**
# Description

- Subscriptions to notifications without admin approval
- Send separate functionality
- Saving Distribution List queries as well as static lists
- Sending resources data (parent-child) in the email
- Add filter grouping
- Sending reference data tables in the email

## Useful links

- Frontend PR: [Frontend](https://github.com/ReliefApplications/ems-frontend/pull/2646)

## Type of change

- New feature 
- Improvement

## Issues Pending
- Send Individual blocks' email fields must also be in dataset fields (in development)

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -27,6 +27,15 @@ MAIL_FROM_PREFIX=
 MAIL_USER=
 MAIL_PASS=
 
+# AZURE FUNCTION MAIL CONFIGURATION
+MAIL_SERVERLESS_URL=
+MAIL_SERVERLESS_KEY=
```
Comment (AntoineRelief) [inline .env.dist:32]: @morgang-adappt 
what are the values for that?

Comment (morgang-adappt) [reply .env.dist:32]: The PR has scaffolding for the Azure Function (an endpoint to proxy requests, variable keys added to `.env.dist`, and required changes made in `custom-environment-variables.js`, but the function is not stable enough.

I'd be happy to drop the commits until the function deployment if this is preferable. 

Additionally, how should we communicate these values once the Azure Function is complete? Do we just need to add the values to the App Service's environment variables?

Comment (AntoineRelief) [reply .env.dist:32]: hey @morgang-adappt 

perhaps we can drop them yes, so we're sure about what we include

about the values, we'll add them to the azure service yes
once the function is complete, we can check together

Comment (morgang-adappt) [reply .env.dist]: Hey @AntoineRelief ,

I've pushed a commit the removes all the discussed functionality.

File: .env.dist [hunk 2]
```diff
@@ -27,6 +27,11 @@ MAIL_FROM_PREFIX=
 MAIL_USER=
 MAIL_PASS=
 
+MAIL_BLOB_STORAGE_URL=
```
Comment (AntoineRelief) [inline .env.dist:30]: @morgang-adappt 

similar question for this
It appears it's currently used
but do you have more information about that? What values can I use?

Comment (morgang-adappt) [reply .env.dist:30]: Hi @AntoineRelief,

These are the values used to persist images linked in the email (header, footer, and banner logos). At the moment we have been using a container in the ADAPPT-POC RG with container level access. From looking at the storage accounts in the EMSSAFE-D RG, it seems it should be possible to create an `email-images` (or similar) container for use, and use the appropriate endpoint, account name, access key, and container name.

**Comment by AntoineRelief** [comment]
@morgang-adappt 

closing the PR as I integrated the work in` beta-unstable` branch

I'll put more comments about the work in the front-end PR after closing it

**Pull request closed without merging**